string model_name
---
string message
string[] name
geometry_msgs/Vector3[] min_bounds
geometry_msgs/Vector3[] max_bounds
geometry_msgs/Vector3[] dimensions
geometry_msgs/Pose[] pose